[MM-61]: Added the test cases for reopened event on the Github and excluded repositories in an organization. (#154)
mattermost/mattermost-test-management: data/test-cases/plugins/github/subscriptions/Exclude_subscriptions_for_repo.md

---
# (Required) Ensure all values are filled up

> name: "Excluding subscription in the channel or DM/GM on MM for a repository in a organization "

Reviewer: ```suggestion
name: "Excluding subscription in the channel or DM/GM for a repository in an organization"
```

cypress: null
detox: null
mmctl: null
playwright: null
rainforest: []
manual_test_environments: []

# Do not change
id: null
key: null
created_on: null
last_updated: null
case_hashed: null
steps_hashed: null
---

**Step 1**

1. Enter the slash command `/github subscriptions add <organization> --feature <events> --exclude <repository>` in the desired channel or DM/GM on MM and create a subscription for an organization with more than one repository and exclude any desired repository in that organization.
2. Trigger any desired event in the excluded repository on the Github and navigate to the desired channel or DM/GM on MM.

**Step 2**

1. Enter the slash command `/github subscriptions add <organization> --feature <events> --exclude <repository>` in the desired channel or DM/GM on MM and create a subscription for an organization with more than one repository and exclude any desired repository in that organization.
2. Trigger any desired event in any desired repository except the excluded ones on the Github and navigate to the desired channel or DM/GM on MM.

**Expected**

No notification should be generated for the triggered event in the desired channel or DM/GM on MM.

> After step 2, notification of the triggered event should be shown in the desired channel or DM/GM on MM.

Reviewer: Please organize following the step/expected format as commented in the other PR.
